johnsonandjohnson/Bodiless-JS
# feat(cli): Create CLI to install local packages

## Changes
- Adds a "bodiless" cli package
- Adds a command to the bodiless cli which installs packages from a local monorepo to a local site.

## Test Instructions
- Clone this branch of the monorepo
- Build and pack the bodiless cli tool - install it globally
- `npm run new`
- Change to new site directory
- Use bodiless tool to isntall one or more packages from monorepo (ideally, modify a package so you can identify that it has changed)
- Verify that the local version of the package was installed

## Related Issues
Fixes #187
